### PR TITLE
fix: wc menu keyboarding and aria-expanded

### PR DESCRIPTION
**Pull request by scomea**
## Description
A timing issue was causing the menu to check the role attribute of children to determine focusability before those children had a chance to update their roles.  Waiting to check on the value change callback resolved that. Also, we were always setting an "aria-expanded" attribute on menu items which would cause menu items to announce as not expanded even when they had no expanded state to begin with.

## Motivation & context
partially fixes https://github.com/microsoft/fast/issues/3481

## Issue type checklist
- [ ] **Chore**: A change that does not impact distributed packages.
- [x] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**
- [ ] I have added a new component
- [x] I have modified an existing component
- [x] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/web-components/fast-foundation/src/menu-item/menu-item.ts [hunk 1]
```diff
@@ -34,10 +34,10 @@ export class MenuItem extends FASTElement {
      *
      * @public
      * @remarks
-     * HTML Attribute: aria-expanded
+     * HTML Attribute: expanded
      */
-    @attr({ attribute: "aria-expanded", mode: "reflect", converter: booleanConverter })
-    public expanded: boolean = false;
+    @attr({ attribute: "expanded" })
+    public expanded: boolean;
```
Comment (chrisdholt) [inline packages/web-components/fast-foundation/src/menu-item/menu-item.ts:40]: This needs to be `aria-expanded` or AT will not recognize it as expanded. The attr should stay the same here. I think we you'll want is to keep the boolean attribute converter but change "reflect" to "fromView".

Comment (scomea) [reply packages/web-components/fast-foundation/src/menu-item/menu-item.ts:40]: We write to the "aria-expanded" attribute in the template, just like we do for disabled:
aria-expanded="${x => x.expanded}"

Comment (scomea) [reply packages/web-components/fast-foundation/src/menu-item/menu-item.ts:40]: If it's undefined we don't render a value, which fixes the bug

Comment (chrisdholt) [reply packages/web-components/fast-foundation/src/menu-item/menu-item.ts:40]: gotcha, thanks for the additional context.

**Pull request merged**